config/routes.rb

```diff
@@ -50,4 +50,7 @@
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
+
+  get '/user/password/edit', to: 'users_password#edit'
```

Comment:
the `user_password` controller takes care of the password update and keeps our routes ReSTful